src/vm/gdbjit.cpp

```diff
@@ -920,7 +1147,7 @@ void NotifyGdb::MethodCompiled(MethodDesc* MethodDescPtr)
 
     elfFile.MemPtr.SuppressRelease();
 
-#ifdef GDBJIT_DUMPELF
+#if 1//def GDBJIT_DUMPELF
```

Comment:
This seems like a forgotten change made during debugging that should have been reverted.